spec/rspec/support/object_formatter_spec.rb

```diff
@@ -158,6 +158,13 @@ def with_delegate_loaded
             matcher_without_a_description.inspect)
         end
       end
+
+      context 'with truncation' do
+        it 'produces an output of limited length' do
+          formatter = ObjectFormatter.new(10)
```

Comment:
I'd like to see a length of `-1` disable this behaviour, and print the full object. Could you implement and test that. Other than that, I think this is good to merge. Also: we need a test for an object with an inspect string that is shorter than this number. What happens here with a 9 character string?